docs/Concepts/Privacy/PrivateTransactionManager.md

```diff
@@ -11,52 +11,158 @@ The Private Transaction Manager has two distinct components:
+
+    !!! example "Example ipc-config-file.toml"
+
+        ```toml
+        socket = "tm.ipc"
+        workdir = "path/to/ipc/file"
+        timeout = 5
+        dialTimeout = 1
+        ```
+
+        Where:
+
+        * optional `timeout` is overall timeout when sending messages (seconds), zero disables timeout, default = 5 seconds
+        * optional `dialTimeout` is timeout for connecting to the socket (seconds), default = 1 second
+
+        !!! note
+
+            You can increase `timeout` if transaction manager responses are too slow.
+            `dialTimeout` rarely need to be changed and can work with the default values.
+
+=== "HTTP connection"
+
+    This allows quorum to connect to a private transaction manager using http.
+    This should only be used for development purposes, due to a lack of security on the connection.
+    For production environments, you should enable TLS on the connection, as described in the next section.
+
+    !!! example "Example http-config-file.toml"
+
+        ```toml
+        httpUrl = "HTTP://127.0.0.1:9101"
+        timeout = 5
+        httpWriteBufferSize = 4096
+        httpReadBufferSize = 4096
+        ```
+
+        Where:
+
+        * optional `timeout` is overall timeout when sending messages (seconds), zero disables timeout, default = 5 seconds
+        * optional `writeBufferSize` is size of the write buffer (bytes), if zero or unspecified then uses http.Transport default
+        * optional `readBufferSize` is size of the read buffer (bytes), if zero or unspecified then uses http.Transport default
+
+=== "HTTP connection using TLS"
+
+    Using an `https` url, and changing the `tls` flag to `STRICT` in the HTTP config, will enable TLS encryption over the connection.
+
+    !!! example "Example http-config-file.toml"
+
+        ```toml
+        httpUrl = "HTTPS://127.0.0.1:9101"
+        tlsMode = "STRICT"
+        tlsRootCA = "/path/to/ca-root.cert.pem"
+        tlsClientCert = "/path/to/client-ca-chain.cert.pem"
+        tlsClientKey = "/path/to/client.key.pem"
+        timeout = 5
+        httpIdleConnTimeout = 10
+        httpWriteBufferSize = 4096
+        httpReadBufferSize = 4096
+        ```
+
+        Where:
+
+        * `tlsRootCA` is any combination of comma separated files/directories containing root CA certificates, defaults to host's certificates
+        * `tlsClientCert` is file containing client certificate
+        * `tlsClientKey` is file containing client certificate private key
+
+## Using command line parameters
+
+The command line parameters listed below can be used to configure the connection to the Transaction Manager.
+Note that these can be used in conjunction with the other options given above,
+in which case the command line parameters will override any others.
+
+```text
+QUORUM PRIVATE TRANSACTION MANAGER OPTIONS:
```

Comment:
A proper markdown formatting would make this more readable. I guess it's a copy of the CLI help output, but if it doesn't change so often, the benefit of having a nice format against having to update by spotting changes is in favor of nice format IMO.

Reply:
Amended. Let me know if this format looks ok.